triagebot.toml

```diff
@@ -691,6 +679,41 @@ message_on_remove = "Issue #{number}'s nomination has been removed. Thanks all f
 message_on_close = "Issue #{number} has been closed. Thanks for participating!"
 message_on_reopen = "Issue #{number} has been reopened. Pinging @*T-types*."
 
+[notify-zulip."beta-nominated".compiler]
```

Comment:
I know this isn't the best location to report this (but I just remembered and will be afk soon), IIRC these don't trigger if the PR author has already added <kbd>{beta,stable}-nominated</kbd> *before* actually opening/submitting the PR.

I first noticed this here: https://github.com/rust-lang/rust/pull/137899. I know it's hard to tell but I had to remove and readd the label as you can see here https://github.com/rust-lang/rust/pull/137899#issuecomment-2698402869 for it to truly trigger. Well, you'd need to compare time & date to verify.

Once I'm back home again, I'll open a triagebot issue or even PR.

Reply:
thanks @fmease for the head-up. if you don't get around to sending a patch, let me know. I can try to do that (and maybe ask you to review)